### PR TITLE
Update import_cli.adoc

### PR DESCRIPTION
**Pull request by hanqiuzh**
Issue: https://github.com/open-cluster-management/backlog/issues/3247

Updates:
- updated manual import for new api changes

Note: there was instructions about creating separate image pull secrets for the namespace. I removed it as currently managedcluster-import-controller can only use default image pull secrets.

### CODE REVIEW

File: manage_cluster/import_cli.adoc [hunk 1]
```diff
@@ -47,101 +47,79 @@ oc login
...
-* The controller uses information in `EndpointConfig` to generate a secret named `+${CLUSTER_NAME}-import+`.
-* The `+${CLUSTER_NAME}-import+` secret contains the `import.yaml` that the user applies to a managed cluster to install `multicluster-endpoint`.
+[#importing-the-klusterlet]
 
-[#importing-the-multicluster-endpoint]
-== Importing the multicluster-endpoint
+== Importing the klusterlet
 
-. Obtain the `endpoint-crd.yaml` that was generated by the cluster controller.
+IMPORTANT: The import command contains pull secret information that is copied to each of the imported clusters.
```
Comment (hanqiuzh) [inline manage_cluster/import_cli.adoc:111]: moved warning of pull secret here, and removed steps for creating customized image pull secret as managedcluster-import-controller doesn't support image pull secret configuration for 2.0

File: manage_cluster/import_cli.adoc [hunk 2]
```diff
@@ -47,101 +47,79 @@ oc login
...
   name: <cluster_name>
   namespace: <cluster_name>
 spec:
+  clusterName: <cluster_name>
+  clusterNamespace: <cluster_name>
   applicationManager:
     enabled: true
+  certPolicyController:
+    enabled: true
   clusterLabels:
```
Comment (hanqiuzh) [inline manage_cluster/import_cli.adoc:85]: do we still want to keep these clusterLabels?

Comment (hanqiuzh) [reply manage_cluster/import_cli.adoc:85]: but they are required in crd 😂

Comment (hanqiuzh) [reply manage_cluster/import_cli.adoc:85]: or shall we set them to `<cloud>` `<vendor>`

**Review by leena-jawale — APPROVED** [review]

File: manage_cluster/import_cli.adoc [hunk 3]
```diff
@@ -45,116 +45,95 @@ oc login
 +
 ----
 oc new-project ${CLUSTER_NAME}
+oc label namespace ${CLUSTER_NAME} cluster.open-cluster-management.io/managedCluster=${CLUSTER_NAME}
```
Comment (hanqiuzh) [inline manage_cluster/import_cli.adoc:48]: step added for https://coreos.slack.com/archives/GUGB0LDDL/p1594755686006700
it's for UI optimization, not sure if I should add another step for it with description?

**Review by KevinFCormier — APPROVED** [review]

**Review by TheRealHaoLiu — APPROVED** [review]

**Review by cadawson — COMMENTED** [review]
Thanks. Looks good.

**Pull request merged**